### DISCUSSION TITLE
Change Browser Row Height

### DISCUSSION THREAD

**Discussion by oabdelhalim** [iterate-ch/cyberduck#16009] [Q&A]
Hi,

How can I change the height/line-height of browser rows?
Version 8.9.0 (41543)

Thanks,

**Comment by dkocher** [comment]
You can do so by adjusting the [browser font size](https://docs.cyberduck.io/cyberduck/browser/#adjust-the-browser-font-size).

**Reply by oabdelhalim** [reply]
> You can do so by adjusting the [browser font size](https://docs.cyberduck.io/cyberduck/browser/#adjust-the-browser-font-size).

I already changed the font size but it makes the font very big and the vertical space is almost nothing, so i need to keep the font size to 16px and change the height of the row only to keep a space between the rows.
